engine/pom.xml

```diff
@@ -76,6 +76,21 @@
       <artifactId>zeebe-db</artifactId>
     </dependency>
 
+    <dependency>
+      <groupId>io.camunda</groupId>
+      <artifactId>zeebe-scheduler</artifactId>
+    </dependency>
+
+    <dependency>
+      <groupId>io.camunda</groupId>
+      <artifactId>zeebe-stream-platform</artifactId>
+    </dependency>
+
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-databind</artifactId>
+    </dependency>
+
```

Comment:
:thought_balloon: These dependencies were used in the engine module but not declared since we'd disabled failing the plugin previously. This was removed since the plugin was fixed, and the missing deps were added.